pkg/repo/secondary.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package repo
+
+import (
+	"io/ioutil"
+	"os"
+	"time"
+
+	"sigs.k8s.io/yaml"
+
+	"helm.sh/helm/internal/urlutil"
```

Comment:
Helm moved to Go modules. That means the major version needs to be in the import path. This applies when something has a major version of 2 or higher. So, the helm import paths need to have a /v3 in them. For example,

`"helm.sh/helm/internal/urlutil"` -> `"helm.sh/helm/v3/internal/urlutil"`

Reply:
@mattfarina thanks a lot for reviewing this PR! Fixed by now.